src/extension.ts

```diff
@@ -144,9 +144,11 @@ export function activate(context: vscode.ExtensionContext) {
 				await vscode.commands.executeCommand("cosmosDB.refresh");
 			}
 		});
-	registerCommand('cosmosDB.api.getConnectionString', getConnectionString);
-	registerCommand('cosmosDB.api.getDatabase', getDatabase);
-	registerCommand('cosmosDB.api.revealTreeItem', revealTreeItem);
+
+	return {
```

Comment:
Please merge with master to address the merge conflicts here. The `apiVersion` on the new object should be "1.0.0"